pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -73,6 +76,16 @@ data:
         #   aws_secret_access_key=<REDACTED>
 ```
 
+After you have configured the `ConfigMap` with **MinIO**, OR **Amazon S3**, patch the Velero `AppDeployment` by adding the `configOverrides` value. This applies the `ConfigMap` to your instance after the cluster has been configured:
```

Comment:
```
After you have configured the `ConfigMap`
```
I would change this becuase `ConfigMap` is not configured, but rather created or updated. You can configure and `AppDeployment` by creating or updating a `ConfigMap`.